src/SQLite.cs

```diff
@@ -2806,7 +2862,7 @@ public static IEnumerable<IndexedAttribute> GetIndices (MemberInfo p)
 				 .Select (x => (IndexedAttribute)InflateAttribute (x));
 		}
 
-		public static int? MaxStringLength (PropertyInfo p)
+		public static int? MaxStringLength (MemberInfo p)
```

Comment:
Again, just for API, please add the `PropertyInfo` overload back.

Reply:
Added below